packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -244,6 +246,38 @@ QUnit.module('General', () => {
 
         assert.notOk($items.eq(3).hasClass(FOCUSED_DISABLED_PREV_TAB_CLASS), 'The fourth item does not have specific class');
     });
+
+    QUnit.test('the tabs element must have a horizontal class by default', function(assert) {
+        const $element = $('#tabs').dxTabs({
+            items: [1, 2, 3],
+        });
+
+        assert.ok($element.hasClass(TABS_HORIZONTAL_CLASS));
+    });
+
+    QUnit.test('the tabs element must have a vertical class if orientation is vertical', function(assert) {
+        const $element = $('#tabs').dxTabs({
+            items: [1, 2, 3],
+            orientation: 'vertical',
+        });
+
+        assert.ok($element.hasClass(TABS_VERTICAL_CLASS));
+    });
+
+    QUnit.test('the tabs element must have a correct orientation class in runtime change', function(assert) {
+        const $element = $('#tabs').dxTabs({
+            items: [1, 2, 3],
+        });
+        const tabs = $element.dxTabs('instance');
+
+        assert.ok($element.hasClass(TABS_HORIZONTAL_CLASS));
+
+        tabs.option('orientation', 'vertical');
+        assert.ok($element.hasClass(TABS_VERTICAL_CLASS));
```

Comment:
There is no assert that other class was removed from element

Reply:
Fixed [[here](116ba1d0d6e538ade3847fc0196b07a792bc5990)].